### PR TITLE
CLOUDP-70062: mongocli ops-manager admin backup blockstore(s) lists

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-70062

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
  ./bin/mongocli om admin backup blockstore list -P ops -o json
{
  "links": [
    {
      "rel": "self",
      "href": "http://localhost:8080/api/public/v1.0/admin/backup/snapshot/mongoConfigs?pageNum=1\u0026itemsPerPage=100"
    }
  ],
  "results": [
    {
      "id": "blockstore",
      "uri": "mongodb://test1:test1@localhost:22222",
      "ssl": false,
      "assignmentEnabled": true,
      "encryptedCredentials": false,
      "loadFactor": 1
    }
  ],
  "totalCount": 1
}

```
```
./bin/mongocli om admin backup blockstore list -P ops
ID           URI                                     SSL     LOAD FACTOR
blockstore   mongodb://test1:test1@localhost:22222   false   1
```

### CODE REVIEW

File: internal/cli/opsmanager/backup/backup.go [hunk 1]
```diff
@@ -34,6 +35,7 @@ func Builder() *cobra.Command {
 		CheckpointsBuilder(),
 		EnableBuilder(),
 		config.Builder(),
+		blockstore.Builder(),
```
Comment (gssbzn) [inline internal/cli/opsmanager/backup/backup.go:38]: [nit] big thing here, according to spec this should be `mongocli ops-manager admin backup blockstore(s) lists`
notice the admin part on the command, also make sure that this command is only available for Ops Manager and not Cloud Manager

**Review by gssbzn — CHANGES_REQUESTED** [review]
just one thing that this command namespace is not right, we should have a new admin scope and add it there

File: internal/cli/opsmanager/backup/backup.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/opsmanager/backup/backup.go:38]: ah okay, good call 👍

File: internal/cli/opsmanager/admin/backup/description.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package backup
+
+const (
+	backup = "Manage backup administration for your project."
```
Comment (gssbzn) [inline internal/cli/opsmanager/admin/backup/description.go:17]: here and admin description these are global settings and not per project so we can drop that part

**Review by gssbzn — CHANGES_REQUESTED** [review]
one small nit to drop project from some descriptions as these are global settings

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**